test/e2e/util.go

```diff
@@ -20,16 +20,13 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/wait"
 
-	cco "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
-
 	"github.com/aws/aws-sdk-go-v2/service/wafv2"
 	wafv2types "github.com/aws/aws-sdk-go-v2/service/wafv2/types"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	albo "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
```

Comment:
nit: Normally openshift imports come before k8s and others, and the others are alphabetized.

Reply:
It is an OpenShift style thing, but you have to admit the alphabetized list is super helpful.